### PR TITLE
[WS] BPLAT-11696 Update User Profiles API specs to include age limit

### PR DESCRIPTION
**Pull request by sandeshms**
https://jira.aminocom.com/browse/BPLAT-11696

### CODE REVIEW

File: rest/snippets/UserConfiguration/UserProfile.yml [hunk 1]
```diff
@@ -17,4 +17,8 @@ UserProfile:
     is_child_profile:
       type: boolean
       example: false
-      description: If this flag is set to "true" then the profile will be blocked from viewing adult content
\ No newline at end of file
+      description: If this flag is set to "true" then the profile will be blocked from viewing adult content
+    age_limit:
```
Comment (janiluuk) [inline rest/snippets/UserConfiguration/UserProfile.yml]: a dumb question;
why is the age limit itself associated with user profile?
shouldn't we be matching the profile against something else that has age limit?

Comment (sandeshms) [reply rest/snippets/UserConfiguration/UserProfile.yml]: I wasn’t there for the grooming. @tlangens should know why it was decided to add it per profile.

Comment (tlangens) [reply rest/snippets/UserConfiguration/UserProfile.yml]: The age limit is compared to the content ratings of content. If content rating age > age limit, then you can't view the content.
is_child_profile (which I'd say is really bad naming) just specifies whether to filter out adult entertainment content, regardless of age. Some people just don't want naughty content to be shown with the regular stuff.

Comment (janiluuk) [reply rest/snippets/UserConfiguration/UserProfile.yml]: Ok, i guess this works out on bigger picture. We'll just have to implement the age_limit checking on the logic wrapping the model.

**Pull request merged**